### PR TITLE
合并两个整理信息

### PR DESCRIPTION
**Pull request by nudtpcm**
add some describe
Signed-off-by: nudtpcm <maph9916@163.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# pcm
\ No newline at end of file
+# pcm
+
+这里将会维护一个实验室项目
+
+主要是三横一纵的补充
```
Comment (penghuima) [inline README.md]: 三横一纵描述如下：

**Review by penghuima — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,5 @@
-# pcm
\ No newline at end of file
+# pcm
+
+这里将会维护一个实验室项目
```
Comment (penghuima) [inline README.md:3]: 具体内容

**Review by penghuima — COMMENTED** [review]
what's your pr

**Review by penghuima — APPROVED** [review]
this is what

**Comment by nudtpcm** [comment]
> this is what

**Review by penghuima — CHANGES_REQUESTED** [review]
dtd t

File: README.md [hunk 3]
```diff
@@ -1 +1,5 @@
-# pcm
\ No newline at end of file
+# pcm
+
+这里将会维护一个实验室项目：pcm
```
Comment (penghuima) [inline README.md]: next

**Review by penghuima — APPROVED** [review]

**Review by penghuima — APPROVED** [review]

**Pull request merged**